requirements_test.txt

```diff
@@ -10,5 +10,6 @@ pytest-timeout
 freezegun
 tomli
 aioresponses
+backports.zoneinfo; python_version<"3.9"
```

Comment:
This is only needed (even after https://github.com/zigpy/zigpy/pull/1464), because we're still running GH CI for 3.8, right?

Reply:
Ooo, good catch. This is due to a bad merge.